src/domain/signupGroup/__tests__/validation.test.ts

```diff
@@ -417,43 +367,29 @@ describe('getContactPersonSchema function', () => {
     ).toBe(true);
   });
 
-  test('should return false if email is missing', async () => {
-    expect(
-      await testContactPersonSchema(registration, signups, {
-        ...validContactPerson,
-        email: '',
-      })
-    ).toBe(false);
-  });
-
-  test('should return false if email is invalid', async () => {
-    expect(
-      await testContactPersonSchema(registration, signups, {
-        ...validContactPerson,
-        email: 'user@email.',
-      })
-    ).toBe(false);
-  });
-
-  test('should return false if phone number is missing', async () => {
-    expect(
-      await testContactPersonSchema(registration, signups, {
-        ...validContactPerson,
-        phoneNumber: '',
-        notifications: [NOTIFICATIONS.SMS],
-      })
-    ).toBe(false);
-  });
-
-  test('should return false if phone number is invalid', async () => {
-    expect(
-      await testContactPersonSchema(registration, signups, {
-        ...validContactPerson,
-        phoneNumber: 'xxx',
-        notifications: [NOTIFICATIONS.SMS],
-      })
-    ).toBe(false);
-  });
+  const testCases: [Partial<ContactPersonFormFields>][] = [
+    [{ email: '' }],
+    [{ email: 'not-email' }],
```

Comment:
Here too max length test cases for email and phone number would be fitting additions

Reply:
Added test cases for phone number and email max lengths